content/panorama/scripts/custom_hud.js

```diff
@@ -241,6 +241,14 @@ function HookPanoramaPanels() {
 }
 
 
+// On Death
+function OnDeath(data) {	
+	if(data.entindex_killed === SafeGetPlayerHeroEntityIndex(Game.GetLocalPlayerID()))
```

Comment:
Add space after if

Reply:
Also `if (data.entindex_killed !== SafeGetPlayerHeroEntityIndex(Game.GetLocalPlayerID())) return;`
and dedent the following code, otherwise that condition affects only the first statement